### PR TITLE
Add BrowserLeaks for IP lookup

### PR DESCRIPTION
**Pull request by codenyte**

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -223,6 +223,8 @@ function lookup(hostname, address) {
 			break;
 		case 2:
 			url = `https://www.ip2location.com/${address}`;
+		case 3:
+			url = `https://browserleaks.com/ip/${address}`;
 			break;
```
Comment (tdulcet) [inline popup.js]: You would need to add a `break` statement after case 2 to prevent it from falling through.

**Review by tdulcet — CHANGES_REQUESTED** [review]
Thank you for the PR! This just needs a minor fix and it should be ready to merge.

File: popup.js (hunk 1 above)
Comment (codenyte) [reply popup.js]: Oh sorry, I missed that.

**Review by tdulcet — APPROVED** [review]
Thanks!

**Comment by codenyte** [comment]
Thanks for creating the awesome (FOSS) extension in the first place. I used to use Flagfox, but stopped using it because of privacy concerns. Recently, I discovered Server-Status and I'm really happy with it. More people should know about this.

**Comment by tdulcet** [comment]
Thank you for your kind words! I am glad this extension is useful for you. I did post about it over on [the Flagfox forum](https://flagfox.net/viewtopic.php?f=3&t=784), but did not receive any responses. Anyway, I am planning to publish a new version in the next few weeks, which will include your changes along with some other improvements.

**Pull request merged**